api.raml

```diff
@@ -55,6 +55,10 @@ schemas:
       {
         "$schema": "http://json-schema.org/draft-03/schema",
         "properties": {
+            "id": {
```

Comment:
I think we actually will want to continue hiding our internal `id` primary key (since it's only used for things like foreign key relations). Since there are multiple different systems, many of them with databases, the `employee_id` is how we are uniquely and consistently identifying users across systems.